### ISSUE TITLE
Why return N*2 size array when model.predict(prediction_type='LogProbability') in CatBoostRegressor?

### ISSUE THREAD

**Issue by byuns9334** [catboost/catboost#2040]
Problem: I don't understand why CatBoostRegressor model.predict returns N*2 array. So I'm trying out this code: 

from catboost import Pool, CatBoostRegressor

train_data = [[0, 3, 4],
              [4, 1, 5],
              [8, 1, 2],
              [9, 1, 3]]

train_labels = [2, 0, 1, 1]

eval_data = [[2, 1, 1],
             [3, 1, 2],
             [9, 0, 3],
             [5, 3, 4]]

eval_labels = [0, 1, 2, 4]

eval_dataset = Pool(eval_data,
                    eval_labels)

model = CatBoostRegressor(learning_rate=0.03,
                           loss_function='RMSE')

model.fit(train_data,
          train_labels,
          eval_set=eval_dataset,
          verbose=False)

x = model.predict([eval_data[3]], prediction_type='Probability') 
y = model.predict([eval_data[3]])
print(x)
print(y)

And it prints: 

[[0.17805031 0.82194969]] # for x
[1.52961306] #for y 

So I don't understand why array 'x' is shaped Nx2. I thought it's supposed to be Nx1 where x[i] is probability value for eval_data[i] to be y[i]. Why does x[i] have length 2? What exactly do x[i][0] and x[i][1] mean? Can anybody please give detailed description?

catboost version: 1.0.4
Operating System: Linux
CPU: not related to this issue
GPU: not using

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
`prediction_type='Probability'` means only that CatBoost should apply sigmoid to the predicted values.
Generally, this prediction_type has little sense in regression models.

To estimate the uncertainty of predicted values, RMSEWithUncertainty may be used.
https://catboost.ai/en/docs/concepts/loss-functions-regression#RMSEWithUncertainty

Please feel free to reopen, should you have further questions.
